src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-router.ts

```diff
@@ -7,14 +7,19 @@ import { HomeAssistant } from "../../../../../types";
 import { navigate } from "../../../../../common/navigate";
 import { PageNavigation } from "../../../../../layouts/hass-tabs-subpage";
 
-import { mdiServerNetwork } from "@mdi/js";
+import { mdiServerNetwork, mdiFormatListBulleted } from "@mdi/js";
 
 export const configTabs: PageNavigation[] = [
   {
     translationKey: "ui.panel.config.zwave_js.navigation.network",
     path: `/config/zwave_js/dashboard`,
     iconPath: mdiServerNetwork,
   },
+  {
+    translationKey: "ui.panel.config.zwave_js.navigation.logs",
+    path: `/config/zwave_js/logs`,
+    iconPath: mdiFormatListBulleted,
```

Comment:
Use the same icon we use in other place for logs
```suggestion
    iconPath: mdiMathLog,
```